packages/orbit-components/src/SocialButton/index.d.ts

```diff
@@ -10,14 +10,7 @@ declare module "@kiwicom/orbit-components/lib/Button";
 
 type Type = "apple" | "facebook" | "google" | "twitter";
```

Comment:
According to the [Flow counterpart](https://github.com/kiwicom/orbit/blob/809b0c34f888a411752896c87ffe759284c249ed/packages/orbit-components/src/SocialButton/index.js.flow#L9) this needs to be exported as well, no?